config.toml

```
@@ -123,10 +123,17 @@ languageName ="Deutsch"
 contentDir = "content/de"
 weight = 9
 
+[languages.ur]
+title = "کلاؤڈ کی مقامی لغت"
+description = "CNCF کلاؤڈ کی مقامی لغت کا مقصد کلاؤڈ مقامی ایپلی کیشنز کے بارے میں بات کرتے وقت استعمال ہونے والی عام اصطلاحات کے حوالے کے طور پر استعمال کرنا ہے۔"
+languageName ="اردو (Urdu)"
+contentDir = "content/ur"
+weight = 10
+
 [languages.fr]
 title = "Glossaire Cloud Native"
 description = "Le projet de Glossaire CNCF Cloud Natif a pour objectif de référencer les termes courants utilisés dans les différentes applications cloud native."
-languageName ="Français(French)"
+languageName ="Français (French)"
 contentDir = "content/fr"
 weight = 10
```

@sestegra minor issue at this point, but could you modify the following?

```suggestion
weight = 11
```

---

Modification done